lib/models/Quote.js

```diff
@@ -0,0 +1,27 @@
+const pool = require('../utils/pool');
+const fetch = require('cross-fetch');
+
+module.exports = class Quote {
```

Comment:
This should be a service like QuoteService. As we are not interacting with a db here it doesn't make sense to have it in our models